api/repositories/pod_repository_test.go

```diff
@@ -0,0 +1,235 @@
+package repositories_test
+
+import (
+	"context"
+
+	. "code.cloudfoundry.org/cf-k8s-controllers/api/repositories"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("PodRepository", func() {
+	Describe("GetPod", func() {
```

Comment:
I think this should reflect the function name: `FetchPodStatsByAppGUID`.

Reply:
Addressed in commit https://github.com/cloudfoundry/cf-k8s-controllers/pull/248/commits/5a4d14ec897981ac8a15f7962ccf70c663466818